app/model/Event/ReadModel/QueryHandlers/EducationFunctionsQueryHandler.php

```diff
@@ -17,14 +17,22 @@ public function __construct(private Skautis $skautis)
     {
     }
 
-    public function __invoke(EducationFunctions $query): Functions
+    public function __invoke(EducationFunctions $query): EFunctions
     {
         $functions = $this->skautis->event->eventFunctionAllEducationLeader([
             'ID_EventEducation' => $query->getEducationId()->toInt(),
         ]);
 
-        return new Functions(
-            ...array_map([PersonFactory::class, 'create'], $functions),
+        $assistants = $this->skautis->event->eventFunctionAllEducationAssistant([
+            'ID_EventEducation' => $query->getEducationId()->toInt(),
+        ]);
+
+        return new EFunctions(
+            PersonFactory::create($functions[0]),
+            PersonFactory::create($functions[1]),
+            PersonFactory::create($functions[2]),
+            PersonFactory::create($functions[3]),
```

Comment:
Tohle (stejně jako předtím) spoléhá na to, že IS ty funkce vrací vždy ve stejném pořadí...